templates/zerver/compose.html

```diff
@@ -95,12 +95,12 @@
                   <a class="message-control-button icon-vector-font" title="{{ _('Formatting') }}" data-overlay-trigger="markdown-help"></a>
                   <a id="undo_markdown_preview" class="message-control-button icon-vector-edit" style="display:none;" title="{{ _('Write') }}"></a>
                   <a id="markdown_preview" class="message-control-button icon-vector-eye-open" title="{{ _('Preview') }}"></a>
-                  <a class="drafts-link" href="#drafts">{{ _('Drafts') }}</a>
+                  <a class="drafts-link" href="#drafts" title="{{ _('Drafts') }} (d)">{{ _('Drafts') }}</a>
                   <span id="sending-indicator">{{ _('Sending...') }}</span>
                   <div id="send_controls" class="new-style">
                     <label id="enter-sends-label" class="compose_checkbox_label" for="enter_sends">{{ _('Press Enter to send') }}&nbsp;</label>
                     <input type="checkbox" id="enter_sends" name="enter_sends" value="enter_sends" />
-                    <button type="submit" id="compose-send-button" class="button small send_message" tabindex="150">{{ _('Send') }}</button>
+                    <button type="submit" id="compose-send-button" class="button small send_message" tabindex="150" title="{{ _('Send') }} (Tab then Enter, Ctrl + Enter)">{{ _('Send') }}</button>
```

Comment:
We should probably just document Ctrl+Enter here, since "then" doesn't internationalize.